### DISCUSSION TITLE
Unsupervised and Supervised Machine Learning

### DISCUSSION THREAD

**Discussion by Debaditya-Som** [community/community#62229] [Programming Help]
What is the difference between supervised and unsupervised learning in machine learning?

**Comment by Anushka-Bhowmick** [comment]
1. Supervised Learning:

The algorithm learns from a labeled dataset, where each data point has corresponding input features and known output labels.
During training, the model adjusts its parameters to minimize the error between its predictions and the actual labels.
Supervised learning is used for tasks like classification and regression, where the goal is to map inputs to desired outputs.
Popular algorithms in supervised learning include Support Vector Machines (SVM), Decision Trees, Random Forests, Neural Networks, and Linear Regression.

2. Unsupervised Learning:

The algorithm learns from an unlabeled dataset without explicit output labels, focusing on discovering inherent patterns or structures.
Clustering is a common application of unsupervised learning, where similar data points are grouped together based on their features.
Another application is dimensionality reduction, which reduces the number of input features while retaining important information.
Unsupervised learning is valuable for exploratory data analysis and data preprocessing.
Popular algorithms in unsupervised learning include K-Means Clustering, Hierarchical Clustering, Principal Component Analysis (PCA), and Autoencoders.

**Comment by queenofcorgis** [comment]
@Debaditya-Som @Anushka-Bhowmick GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
